Dart/src/com/jetbrains/lang/dart/ide/documentation/DartDocUtil.java

```diff
@@ -268,7 +269,7 @@ private static void appendClassSignature(final StringBuilder builder, final Dart
     final List<DartType> mixins = dartClass.getMixinsList();
     final DartType superClass = dartClass.getSuperClass();
     if (superClass != null) {
-      builder.append("<br/>extends ").append(StringUtil.escapeXml(superClass.getText()));
+      builder.append(" extends ").append(StringUtil.escapeXml(superClass.getText()));
```

Comment:
Don't you like it on separate lines like done for Java? Implements list may be long.